action.yaml

```diff
@@ -84,9 +84,9 @@ runs:
         GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
       run: |
           # Call the GitHub API to get check runs associated with the push event
+          echo "Brnach name - ${{ github.head_ref || github.ref_name }}"
```

Comment:
If you're going to keep the `echo` then there is a typo in there. Should be `Branch`